lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe 'ЕТИ' do
+  before(:all) do
+    @cs_eti_page = CompanySite::EtiPage.new
+    @cs_main_page = CompanySite::MainPage.new
+
+    log_in_as(:user)
+    navigate_to_eti
+    @cs_main_page.close_banner
+  end
+
+  describe 'Добавление групп' do
+    context 'когда привязываем группу к товару' do
+      before(:all) do
+        @product = {
+            name: Faker::Number.number(5)
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.